pkg/pillar/cmd/zedkube/kubeservice.go

```diff
@@ -0,0 +1,334 @@
+// Copyright (c) 2025 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build kubevirt
+
+package zedkube
+
+import (
+	"context"
+	"net"
+	"sort"
+
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// Define the K8s service CIDR that we want to exclude from external IP handling
+const (
+	kubeServicePrefix = "10.43.0.0/16" // Standard K3s service CIDR
```

Comment:
This could be a global `*net.IPNet` variable instead of string constant to avoid parsing it on every call to `GetAllKubeServices`.

Reply:
Is there a better place to keep all constant addresses in use?

Reply:
will do. thanks.